py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,28 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
```

Comment:
We should build this based on the `format_prefix` in the deprecation stuff.  I think I'd not worry about `format_no_conn_prefix` as otherwise I think maybe this is not possible.